scripts/env.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This file could maybe also be moved to `./scripts/helper`.

Reply:
done